index.html

```diff
@@ -421,6 +421,8 @@
 
 				<!-- House/Senate districts switcher -->
 
+				<p onClick="zoomToPolygon('', '-108,25,-88,37,0', '')" class="zoomoutlink">Show whole state</p>
```

Comment:
If you decide against including the zoom out link, you just have to prune this one line.  And if you want to style it differently, `zoomoutlink` is the CSS class to edit.

Reply:
Thanks for presenting these options. I think we defer to RYHT's preference on this one after they've had a chance to play with v2.0.